### PR TITLE
ci: Move API URL for Clearance UI to repository variable

### PR DESCRIPTION
**Pull request by mmurto**
This allows changing it without creating new commits.

Signed-off-by: Mikko Murto <mikko.murto@doubleopen.org>

### CODE REVIEW

File: .github/workflows/docker.yml [hunk 1]
```diff
@@ -16,6 +16,9 @@ jobs:
     build:
         runs-on: ubuntu-latest
 
+        env:
+            NEXT_PUBLIC_API_URL: ${{ vars.NEXT_PUBLIC_API_URL }}
```
Comment (sschuberth) [inline .github/workflows/docker.yml:20]: Excuse my ignorance, but where are `vars` defined?

Comment (mmurto) [reply .github/workflows/docker.yml:20]: They're defined in https://github.com/doubleopen-project/dos/settings/variables/actions. At least VSCode also provides autocomplete and warnings in the IDE if they're not defined.

Comment (sschuberth) [reply .github/workflows/docker.yml:20]: > At least VSCode also provides autocomplete and warnings in the IDE if they're not defined.

Uh, that means VSCode does GitHub API calls for that?

**Review by sschuberth — APPROVED** [review]

File: .github/workflows/docker.yml (hunk 1 above)
Comment (mmurto) [reply .github/workflows/docker.yml:20]: > > At least VSCode also provides autocomplete and warnings in the IDE if they're not defined.
> 
> Uh, that means VSCode does GitHub API calls for that?

I haven't gone under the hood, but I'm sure [the plugin responsible for that](https://marketplace.visualstudio.com/items?itemName=GitHub.vscode-github-actions) calls the API.

**Pull request merged**